### ISSUE TITLE
image auroc in visa

### ISSUE THREAD

**Issue by NianWenxin** [Mwxinnn/AA-CLIP#4]
在mvtec上full-shot训练，在visa上测试。像素级auroc达到95.03（论文中为95.5）；图像级auroc只有79.04，和论文中所写的84.6差距过大。所使用的jsonl文件是dataset/seed中的。

**Comment by Mwxinnn** [comment]
你好，谢谢你的提醒。
由于我们认为VisA中的异常情况更接近真实情况，我们主要的实验配置是根据VisA调整的。在MVTec-AD的训练中我们使用了稍微不同的参数配置。我们会在后续版本中更新这部分内容。

**Comment by lixinying0727** [comment]
> 在mvtec上full-shot训练，在visa上测试。像素级auroc达到95.03（论文中为95.5）；图像级auroc只有79.04，和论文中所写的84.6差距过大。所使用的jsonl文件是dataset/seed中的。
你好，我想问一下，你测试用了多长时间，我的训练好了VisA的权重，测试MVTec一天一夜了还没出结果，用的4090

**Comment by lzl2040** [comment]
> 你好，谢谢你的提醒。 由于我们认为VisA中的异常情况更接近真实情况，我们主要的实验配置是根据VisA调整的。在MVTec-AD的训练中我们使用了稍微不同的参数配置。我们会在后续版本中更新这部分内容。

你好，什么时候会更新呢，我遇到了跟上面同样的问题，参数配置包括哪些？

**Comment by lzl2040** [comment]
> 在mvtec上full-shot训练，在visa上测试。像素级auroc达到95.03（论文中为95.5）；图像级auroc只有79.04，和论文中所写的84.6差距过大。所使用的jsonl文件是dataset/seed中的。

你好，请问解决这个问题了吗

**Comment by Sword987** [comment]
> > 在mvtec上full-shot训练，在visa上测试。像素级auroc达到95.03（论文中为95.5）；图像级auroc只有79.04，和论文中所写的84.6差距过大。所使用的jsonl文件是dataset/seed中的。
> 
> 你好，请问解决这个问题了吗

同问，感谢~

**Comment by coordxyz** [comment]
> > 你好，谢谢你的提醒。 由于我们认为VisA中的异常情况更接近真实情况，我们主要的实验配置是根据VisA调整的。在MVTec-AD的训练中我们使用了稍微不同的参数配置。我们会在后续版本中更新这部分内容。
> 
> 你好，什么时候会更新呢，我遇到了跟上面同样的问题，参数配置包括哪些？

同问MVTec-AD的训练中具体使用了哪些不同的参数配置？
另，能否分享一下在VisA测试时每类物体上的性能？
谢谢

**Comment by coordxyz** [comment]
> 在mvtec上full-shot训练，在visa上测试。像素级auroc达到95.03（论文中为95.5）；图像级auroc只有79.04，和论文中所写的84.6差距过大。所使用的jsonl文件是dataset/seed中的。

@NianWenxin 请问你试过在visa上full-shot训练，在mvtec上测试吗？性能跟论文一致吗？

**Comment by IAMShashankk** [comment]
> > Full-shot training on MVTEC and testing on VISA. The pixel-level AUROC reached 95.03 (95.5 in the paper); the image-level AUROC was only 79.04, significantly lower than the 84.6 reported in the paper. The jsonl file used is from dataset/seed.
> 
> [@NianWenxin](https://github.com/NianWenxin)Have you tried full-shot training on VISA and testing on MVTEC? Is the performance consistent with the paper?

I tried this, the performance of Image AUROC is not the same.

**Comment by Kerio99** [comment]
meet the same problem~

**Comment by zhoni14** [comment]
> 你好，谢谢你的提醒。 由于我们认为VisA中的异常情况更接近真实情况，我们主要的实验配置是根据VisA调整的。在MVTec-AD的训练中我们使用了稍微不同的参数配置。我们会在后续版本中更新这部分内容。

mvtec的训练集只有正常样本，你打算怎么引入异常样本？从测试集？

**Comment by jeffreyFF** [comment]
复现遇到同样的问题，我真的非常需要在VisA数据集上进行训练的超参数。
I really needI encountered the same issue during reproduction and really need the hyperparameters used for training on the VisA dataset.
